content/docs/config-files.md

```diff
@@ -12,7 +12,16 @@ Notepad++ offers a comprehensive user interface to review or change most of its
 * Editing previously-recorded macros, or crafting new macros manually
 * Adding keywords to a language, because the new language version isn't matched yet
 
-The underlying XML files are all found in `%AppData%\notepad++\` (or, for zip-based local installations, in the notepad++ executable directory) unless otherwise noted.
+## Configuration Files Location
+
+In a standard installation, the configuration files go in `%AppData%\Notepad++\`.  (For a refresher course on `%AppData%`, see the [Community Forum FAQ's `%AppData%` entry](https://community.notepad-plus-plus.org/topic/15740/faq-desk-what-is-appdata).)
+
+In a portable installation, the configuration files go in the same directory as the `notepad++.exe` executable.  (An installation is treated as "portable" if the zero-byte `doLocalConf.xml` file is present alongside the `notepad++.exe` executable.  The **?** menu's **Debug Info** will show `Local mode: ON` for a portable version.)
+
+If you enable the [Cloud settings](../preferences/#cloud), some configuration files will go in the defined directory (including `contextMenu.xml`, `shortcuts.xml`, `userDefineLang.xml`, `langs.xml`, `stylers.xml`, and `config.xml`;
+the `userDefineLang\` subfolder can be placed there as well, though it won't be created by default when the Cloud settings folder is first populated).
+
+If the Cloud directory is enabled, that will take priority over the portable or standard config file location; if `doLocalConf.xml` is present, it will take priority over the `%AppData%\Notepad++\` folder; if neither are active, then the standard config file location is `%AppData%\Notepad++\`.
 

```

Comment:
This is good, but there is new info you have seen probably in:
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/b6a66ba2b110c0d9a190006c5cba013454d40310

In custom settings dir, one more file is including: `session.xml`

Reply:
regarding the new info: I opened that as a separate issue (#141), and I was planning on adding that in using a separate PR once the new version number is added (for new features, I like including the version number, so that people who are a couple versions behind can tell which part of the manual doesn't apply to them.)

Regarding session.xml, I'll add that in.

Reply:
added session.xml to the list.